README.md

```diff
@@ -89,6 +91,8 @@ paySonect: function() {
     // is supposed to process the transaction. Also you should pass the paymentReference obtained  
     // from your server. 
     let paymentReference = {
+        value: "20.00",
```

Comment:
In android we still need value + currency here just because we simply close SDK and send data back to outer app and only it knows what the payment details are.